jupyter/datascience/ubi8-python-3.8/Pipfile

```diff
@@ -0,0 +1,38 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[dev-packages]
+
+[packages]
+# Datascience and useful extensions
+boto3 = "~=1.26.69"
+jupyter-bokeh = "~=3.0.5"
+elyra-python-editor-extension = "~=3.14.2"
+kafka-python = "~=2.0.2"
+matplotlib = "~=3.6.3"
+numpy = "~=1.24.1"
+pandas = "~=1.5.3"
+plotly = "~=5.13.0"
+scikit-learn = "~=1.2.1"
+scipy = "~=1.10.0"
+jupyterlab-lsp = "~=3.10.2"
+jupyterlab-widgets = "~=3.0.5"
+jupyter-resource-usage = "~=0.6.0"
+jupyterlab-s3-browser = "~=0.10.1"
```

Comment:
I see lot of packages are updated in here with respect to requirements.in 
however some packages are downgraded, like jupyterlab-s3-browser from 0.12.0 to 0.10.x 
any particular reason for this? or was it due to dependency resolution. ?

Reply:
Exactly, the downgrade on jupyterlab-s3-browser package happened due to dependency resolution.

Reply:
At some point we should question the inclusion of jupyterlab-s3-browser. It has not been updated since July 2022, the main contributor seems to have switched to something else.  I fear that at some point it will slow/block us, requiring packages that are totally obsolete. @atheo89 solved the issue by downgrading a lot, which removed bad requirements,  I just got rid of it in my contrib images.